app/controllers/home_controller.rb

```diff
@@ -1,4 +1,6 @@
 class HomeController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:index]
```

Comment:
We don't want to skip to authentication here.
The Home controller should only be accessible by authenticated users, no?